### DISCUSSION TITLE
about ThinkPad t590 cannot start the system when replacing the display

### DISCUSSION THREAD

**Discussion by 1074566721** [yusifsalam/t490-macos#45] [General]
**_All fields are mandatory._**

**Describe the bug**
OK, I use the translation software. I don't know if I can express it clearly
 My original monitor is n156hca-eab, and the replaced monitor is n156hce-gn1, both of which are 1080p.
After replacing n156hce-gn1, the system cannot be started again. It can be started normally by replacing the original display. Find a solution.
And，The NumLock key light of t590 is not on (in all versions).

**Hardware**
ThinkPad T590  i5-8525u  mx250 

**Software and firmware**
11.4, 1.73/1.23, 0.6.5.

**To Reproduce**
100%

**Expected behavior**

**Screenshots**
![U1`07LX4M_MUQNSC2B}}$KK](https://user-images.githubusercontent.com/67006196/129667184-ff83a670-aa84-4383-bfc6-68e52796cdd1.jpg)

If you use the latest version of 0.7.1 OpenCORE, the following prompt will appear

![IMG_20210817_104202](https://user-images.githubusercontent.com/67006196/129673273-f80be26b-8386-4f74-8111-21c293e5fcef.jpg)

oc：failed to parse configuration

**Comment by yusifsalam** [comment]
Hi, 

Does everything work normally in Windows or Linux? To be honest, I don't know why macOS wouldn't boot with your new display. You could try to reset NVRAM and see if it helps. 
For NumLock light, you can use YogaSMC to configure the behavior. 
I just merged a PR, can you try if that fixes the OC error? I don't have access to the laptop anymore so I am not able to test myself.

**Comment by 1074566721** [comment]
Let me try this new project.
Everything is normal in windows. I tried to reset NVRAM and couldn't solve the problem. I looked up the online tutorial and tried to inject display EDID code, but I couldn't complete the injection without other macOS computers.

**Comment by 1074566721** [comment]
This is my display EDID code：
00FFFFFFFFFFFF000DAE0C15000000002A1C0104B522137802EE95A3544C99260F505400000001010101010101010101010101010101363680A0703820405036680058C110000018363680A0703820405036680058C110000018000000FE00434D4E0A202020202020202020000000FE004E3135364843452D474E310A2001D1

**Comment by 1074566721** [comment]
I tried the latest project. doesn't boot macOS
![IMG_20210818_191442](https://user-images.githubusercontent.com/67006196/129891341-ba85b63d-d454-4354-966c-691257fb5d47.jpg)

**Comment by yusifsalam** [comment]
I don't really see any errors there. So does the screen go black or is it completely stuck here and nothing happens?

**Comment by yusifsalam** [comment]
Also, have you tried connecting your laptop to an external monitor?

**Comment by 1074566721** [comment]
Is it completely stuck here and nothing happens.
I tried connecting to an external monitor,boot macOS normally.
![6CF0EA245E4E046AED921318728E7F88](https://user-images.githubusercontent.com/67006196/129992053-fd031df9-822b-46d3-9acb-da988bcb0634.jpg)

**Comment by yusifsalam** [comment]
Very interesting. 
To be honest I don't really know what could be causing the new display to not work with macOS but this does not seem like an issue with my config files since both the old display and external monitor work. I'm not able to help any further so I will convert this issue to a discussion, maybe you can get some help from someone else there.
